autoheal.yml

```diff
@@ -62,6 +62,30 @@ awx:
   #
   project: "My project"
 
+#
+# This section describes how to throttle the execution of healing rules.
+throttling:
+  #
+  # The time that the service will remember an executed healing action. If an
+  # action is triggered more than once in the given interval it will be executed
+  # only the first time. The rest of the times it will be logged and ignored.
+  #
+  # Note that for throttling purposes actions are considered the same if they
+  # have exactly the same fields with exactly the same values *after* processing
+  # them as templates.  For example, an action defined like this:
+  #
+  #   awxJob:
+  #     template: "Restart {{ $labels.service }}"
+  #
+  # Will have different values for the `template` field if the triggering alerts
+  # have different `service` labels.
+  #
+  # interval is a duration string ,that is, a sequence of decimal numbers,
```

Comment:
Sorry, but I can't forget the typing classes I took when I was very young :-) . Can you start the sentence with upper case and wrap the lines at 80 characters?

Reply:
Sure!